requirements.txt

```diff
@@ -2,3 +2,4 @@ numpy>=1.16.0
 pandas>=0.25.0
 pvlib>=0.7.0
 scipy>=1.3.0
+ruptures
```

Comment:
May want to make this an optional dependency, check for import in the `shifts_ruptures` function and raise an error if not found.

Reply:
I think that is a good idea.